test/dynamo/test_exceptions.py

```diff
@@ -526,6 +559,260 @@ def fn(t):
         t = torch.randn(2)
         fn(t)
 
+    @unittest.expectedFailure
+    @make_dynamo_test
+    def test_raise_set___context__(self):
+        try:
+            raise TypeError
+        except TypeError as e:
+            exc = e
+
+        self.assertIsNone(exc.__context__)
+
+        try:
+            raise ValueError
+        except ValueError as e:
+            exc2 = e
+
+        self.assertIsNone(exc2.__context__)
+
+
+class CPythonExceptionTests(torch._dynamo.test_case.TestCase):
+    # Tests taken from CPython source code in cpython/Lib/test/test_exceptions.py
+    # https://github.com/python/cpython/blob/v3.13.1/Lib/test/test_exceptions.py
+
+    @unittest.expectedFailure
+    @make_dynamo_test
+    def testChainingAttrs(self):
+        e = Exception()
+        self.assertIsNone(e.__context__)
+        self.assertIsNone(e.__cause__)  # we don't track __cause__
+
+        e = TypeError()
+        self.assertIsNone(e.__context__)
+        self.assertIsNone(e.__cause__)
+
+        class MyException(OSError):
+            pass
+
+        e = MyException()
+        self.assertIsNone(e.__context__)
+        self.assertIsNone(e.__cause__)
+
+    @unittest.expectedFailure
+    @make_dynamo_test
+    def testChainingDescriptors(self):
```

Comment:
For each of the expectedFailure'd tests:
- what's wrong with them?
- will they be resolved in a follow-up PR?

Reply:
Oh, the failures are because of the unittest API. The expectedFailure decorator is removed in a follow-up PR.

Reply:
thanks for clarifying